core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -71,6 +71,22 @@ trait RawlsInstrumented extends WorkbenchInstrumented {
     ExpandedMetricBuilder
       .expand(WorkspaceMetricKey, "entity_cache")
       .asTimer("staleness")
+
+  /**
+    * A gauge to track the number of entities in each cloned workspace.
+    */
+  protected def clonedWorkspaceEntityHistogram: Histogram =
+    ExpandedMetricBuilder
+      .expand(WorkflowStatusMetricKey, "cloned_ws_entities")
+      .asHistogram("count")
+
+  /**
+    * A gauge to track the number of attributes in each cloned workspace.
+    */
+  protected def clonedWorkspaceAttributeHistogram: Histogram =
+    ExpandedMetricBuilder
+      .expand(WorkflowStatusMetricKey, "cloned_ws_attributes")
+      .asHistogram("count")
```

Comment:
1. Update the comments to refer to histograms, not gauges
2. I think these should use `WorkspaceMetricKey` instead of `WorkflowStatusMetricKey`, right?